package.json

```diff
@@ -44,7 +44,8 @@
     "nth-check": "^2.0.1",
     "axios": "^1.6.8",
     "semver": "^7.5.3",
-    "@babel/traverse": "^7.23.2"
+    "@babel/traverse": "^7.23.2",
+    "cross-spawn": "^6.0.6"
```

Comment:
Blir denne brukt?

Reply:
idk, er kanskje to avhengigheter som bruker forskjellig versjoner av cross-spawn? Er i det minste ikke en direkte avhengighet

@jeffreiffers husker du hvorfor du la til denne? (Jeg har basert dette på det du satte opp på fredag)